bin/zowe-setup-keyring-certificates.env

```diff
@@ -0,0 +1,33 @@
+# TODO - do we need a workflow for keyrings?
+
+# The hostname of the system running API Mediation. If the hostname is
+# omitted, the configuration script attempts to calculate the value on 
+# its own. You can specify multiple domains separated by comma.
+HOSTNAME=
+# The IP address of the system running API Mediation. If the IP address 
+# is omitted, the configuration script attempts to calculate the value 
+# on its own. You can specify multiple IPs separated by comma.
+IPADDRESS=
+# Should APIML verify certificates of services it uses - true/false
+VERIFY_CERTIFICATES=true
+
+# If APIML SSO token not present, Zowe components are allowed to attempt authentication with other user-provided data
+SSO_FALLBACK_TO_NATIVE_AUTH=true
+# optional - PKCS#11 token name for SSO. Must already exist
+PKCS11_TOKEN_NAME=
+# optional - PKCS#11 token label for SSO. Must not already exist
+PKCS11_TOKEN_LABEL=
+
+# Location for generated certificates and/or JWT token
+KEYSTORE_DIRECTORY=/global/zowe/keystore
+# This variable has to be set to the Zowe certificate's LABEL specified in the JCL.
+ZOWE_CERTIFICATE_LABEL=localhost
+# Specify zowe user id to set up ownership of the generated certificates.
+# Set the variable to the same user id as in the jcl.
+ZOWE_USER_ID=ZWESVUSR
+# Specify zowe keyring that keeps zowe certificates, set
+# the variable to the same keyring that you used in the jcl.
+ZOWE_KEYRING=
```

Comment:
Would it be better if this defaulted to match the ZWEKRING JCL, so it becomes
`ZOWE_KEYRING=ZoweKeyring`

Reply:
Fixed